packages/amplify-schema-validator/src/__tests__/schemas/valid-owner-field-type-string.graphql

```diff
@@ -0,0 +1,7 @@
+type Inicio
+  @model
+  @auth(
+    rules: [{ allow: owner, operations: [create, delete, update, read], provider: userPools, ownerField: "autores", groups: ["MeuCPD"] }]
```

Comment:
Have you intentionally added `groups` for owner rule? Are there any validations in transformers to stop this?

Reply:
No, one of our customers had this schema. So just used the same schema for our test. The validation on our end only checks for ownerField type
I'm not sure if any validations exists on the data side to stop this.